lib/guardian.ex

```diff
@@ -727,7 +727,7 @@ defmodule Guardian do
   Provides a way to track an invalid return tuple via a nicer reason
   """
 
-  @spec validate_conditional_tuple({:ok, any} | {:error, any} | any) :: {:ok, any} | {:error, any}
+  @spec validate_conditional_tuple({:ok, any} | {:error, any} | any, tuple) :: {:ok, any} | {:error, any}
```

Comment:
Can we update this to be a little more explanatory? `{module, atom}` maybe?

Thanks for catching this!